src/main/java/mate/academy/service/MovieSessionService.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.service;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionService {
+    MovieSession add(MovieSession movieSession);
+
+    MovieSession get(Long id);
```

Comment:
According to the checklist, the 'get' method on the dao layer should return 'Optional', but the service layer should return the entity itself. Please ensure that the corresponding dao method returns 'Optional<MovieSession>' and handle the optional value appropriately in the service layer implementation.